examples/sandbox/index.html

```diff
@@ -117,10 +117,7 @@ <h4>Select</h4>
       <button id="native">Pair Native</button>
       <button id="metaMask">Pair MetaMask</button>
       <button id="phantom">Pair Phantom</button>
-      <button id="xdefi">Pair XDEFI</button>
       <button id="keplr">Pair Keplr</button>
-      <button id="tallyHo">Pair Tally Ho</button>
-      <button id="coinbase">Pair Coinbase</button>
```

Comment:
Removed across the board in this PR:

- Tally was dead for a long time already, and could ironically revive if Tally announced themselves as en EIP-6363 provider https://github.com/tahowallet/extension/issues/3725
- Coinbase doesn't need an explicit button anymore and is handled by the EIP detection
- Same as above for XDEFI

![image](https://github.com/user-attachments/assets/e21e0798-81c9-4dbc-b385-6f11182ea909)